modules/ROOT/pages/microsoft/microsoft-365-ops-connector.adoc

```diff
@@ -183,6 +218,16 @@ payload]]></ee:set-payload>
 </mule>
 ----
 
+Define these attributes in your properties file:
```

Comment:
Is this something that the audience does AFTER pasting the XML syntax? Is this the "experiment" part? Consider a bit more narration in this section to help the bits hold together.